### DISCUSSION TITLE
Appointment Field

### DISCUSSION THREAD

**Discussion by paragonn** [solspace/craft-freeform#716] [Feature Requests]
It would be great to have an appointment field that limits the days/times available for choosing an appointment. Jotform has this feature and it would be amazing to have this feature in Freeform. Please see the screenshot below:
<img width="674" alt="Screenshot 2023-06-26 at 6 18 30 PM" src="https://github.com/solspace/craft-freeform/assets/1228734/cc0533cc-5c40-4e60-a1ce-d0a58cae6a5d">

**Comment by kjmartens** [comment]
Thanks for the feedback, @paragonn! I agree this would be nice to have 🙂
